lib/perl/Genome/Model/Event/Build/ReferenceAlignment/AlignReads.pm

```diff
@@ -488,7 +489,7 @@ sub _process_and_link_alignments_to_build {
     }
 
     for my $alignment (@alignments) {
-        my $link = $alignment->add_user(user => $build, label => 'uses');
+        my $link = $alignment->user(user => $build, label => 'uses');
```

Comment:
If we get here and the user does not exist, there's a bug in the code handling the  `$result_users`.  I am more inclined to remove the existence check here altogether and just print the `Linked alignment ...` debug messages for all alignments.